PULL_REQUEST_TEMPLATE.md

```diff
@@ -22,3 +24,5 @@ Contributors guide: https://github.com/Jasig/uPortal/blob/master/CONTRIBUTING.md
 
 [individual contributor license agreement]: https://github.com/Jasig/uPortal/blob/master/CONTRIBUTING.md#individual-contributor-license-agreement
 [commit guidelines]: https://github.com/Jasig/uPortal/blob/master/CONTRIBUTING.md#commit
+[message properties]: https://github.com/Jasig/uPortal/tree/master/uportal-war/src/main/resources/properties/i18n
+[WCAG 2.0 AA]: https://www.w3.org/WAI/WCAG20/quickref/?levels=aaa&technologies=smil%2Cpdf%2Cflash%2Csl
```

Comment:
The query parameters are filters on the quick reference.
Since uPortal focuses on WCAG A and AA, for HTML, CSS, and ARIA on the client and server.
This filter hides rules from WCAG AAA and for the SMIL, PDF, Flash, and Silverlight technologies which uPortal does not target.
The filter pares down the size of the quick reference to allow contributors to focus on the rules most relevant to them.